astro/src/content/quickstarts/quickstart-go-api.mdx

```diff
@@ -0,0 +1,377 @@
+* Your example customer username is `customer@example.com` and your password is `password`. They will have the role of `customer`.
+* Your admin username is `admin@example.com` and your password is `password`.
+* Your fusionAuthBaseUrl is `http://localhost:9011/`
+
+You can log into the [FusionAuth admin UI](http://localhost:9011/admin) and look around if you want, but with Docker/Kickstart you don't need to.
+
+## Create Your Golang Resource Server Application
+
+Now you are going to create a Golang API application. While this section builds a simple API, you can use the same configuration to integrate an existing API with FusionAuth.
+
+We are going to be building an API backend for a banking application called ChangeBank. This API will have two endpoints:
+- `make-change`: This endpoint will allow you to call GET with a `total` amount and receive a response indicating how many nickels and pennies are needed to make change. Valid roles are `customer` and `teller`.
+- `panic`: Tellers may call this endpoint to call the police in case of an incident. The only valid role is `teller`.
+
+Both endpoints will be protected such that a valid JSON web token (JWT) will be required in the `Authorization` header in order to be accessed. Additionally, the JWT must have a `roles` claim containing the appropriate role to use the endpoint.
+
+<Aside type="note">
+If you simply want to run the application , there is a completed version in the 'complete-application' directory. You can use the following commands to get it up and running if you do not want to create your own.
+
+```shell
+cd complete-application
+go get
+go run main.go
+```
+
+You can then follow the instructions in the Run the API section started at Get a Token.
+</Aside>
+
+To create the application from scratch, make a directory for this API. 
+
+```
+mkdir your-application && cd your-application
+```
+
+### Initialize The Application
+
+Initialize the Golang application using the following:
+
+```
+go mod init your-application
+```
+
+### Write Golang Code
+
+<Aside type="note">
+This tutorial puts all these classes in the same package for simplicity, but for a production application, you'd separate these out.
+</Aside>
+
+Now you are going to write some Golang code. You need to create a file called `main.go`. Here you will add the package clause and import declarations below:
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/main.go"} tags="package-imports"
+  lang="go"/>
+
+You will also need to declare a variable and assign the value of the public key for the application. The value for this key will be set later in the application. Declare this variable below the import declarations.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/main.go"} tags="verify-key"
+  lang="go"/>
+
+### Add Handlers and Functions
+
+Create the main function for the application.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/main.go"} tags="main"
+  lang="go"/>
```

Comment:
Not sure if the `  //this is a newline comment` comment is intentional?

Reply:
not intentional, removed